pkg/catalog/routes.go

```diff
@@ -72,19 +72,18 @@ func (mc *MeshCatalog) listOutboundTrafficPoliciesForTrafficSplits(sourceNamespa
 		}
 		policy := trafficpolicy.NewOutboundTrafficPolicy(buildPolicyName(svc, sourceNamespace == svc.Namespace), hostnames)
 
-		rwc := trafficpolicy.RouteWeightedClusters{
-			HTTPRouteMatch:   wildCardRouteMatch,
-			WeightedClusters: mapset.NewSet(),
-		}
+		weightedClusters := []service.WeightedCluster{}
```

Comment:
maybe define this as a set so that you can avoid the conversion in NewRouteWeightedCluster

Reply:
was done on purpose for this reason: https://github.com/openservicemesh/osm/pull/2892#discussion_r595481448
lmk what you think